madcars/runners/run_original_env.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+
+script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
+agent_folder=$(realpath "$script_dir/../players")
+runner_folder=$(realpath "${RUNNER}")
+num_runs=10
+if [[ ! -z "${1}"  ]]
+then
+    num_runs="${1}"
+fi
+
+echo "------------------------------------"
+echo "agent location: $agent_folder"
+echo "runner location: $runner_folder"
+echo "sessions number: $num_runs"
+echo "------------------------------------"
+echo "Game:"
+
+for ((i=0; i < $num_runs; ++i));
+do
+    python3 $runner_folder/localrunner.py -f "python3 $agent_folder/pytorch_main.py --train" -s "python3 $agent_folder/pytorch_main.py"
```

Comment:
btw, it is advised to have a -u key

Reply:
`-u` is for unbuffered python io. we don't really do any io to the user. I don't think it's necessary. This might cause troubles for localrunner.py reading the output though. Good catch